### ISSUE TITLE
"Terminate batch job" prompt is not answered automatically

### ISSUE THREAD

**Issue by bannmann** [mridgers/clink#307]
The "terminate batch job" prompt is not answered automatically. This happens on Windows 7 (English) with Clink v0.4.4. For example, invoking the `mvn` batch file and pressing Ctrl-C immediately looks like this:
```
Clink v0.4.4 [git:a60c0a] Copyright (c) 2014 Martin Ridgers
http://mridgers.github.io/clink


C:\>mvn
Terminate batch job (Y/N)?
```
I then have to press `y` and hit enter.

**Comment by mridgers** [comment]
Does mvn.bat possibly run other .bat scripts via a child cmd.exe process? Very likely the case if you get a second terminate prompt that does get auto-answered. Consider this simple Batch script;

```
@if "%1"=="" (cmd.exe /c %0 foobar & exit /b 0)
dir /s /b \
```

When run without an argument it launches itself in another cmd.exe and then list the contents of the disk. Clink won't be present in this second cmd.exe process so obviously isn't able to auto-answer any prompts. This is even the case if Clink's autorun is installed as it won't install itself in non-interactive cmd.exe process (e.g. ones with /c on the command line).

**Comment by bannmann** [comment]
No, that batch script does not invoke `cmd`. The behavior can also be observed in a really simple batch file:
````
C:\Users\myuser>echo ping localhost>foo.cmd

C:\Users\myuser>foo.cmd

ping localhost

Pinging mycomputer [::1] with 32 bytes of data:
Reply from ::1: time<1ms
Reply from ::1: time<1ms

Ping statistics for ::1:
    Packets: Sent = 2, Received = 2, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 0ms, Maximum = 0ms, Average = 0ms
Control-C
^CTerminate batch job (Y/N)?
````

The prompt is not auto-answered, and once I answer it, no other "terminate?" prompt appears.

**Comment by mridgers** [comment]
Just to be absolutely certain, you have enabled auto-answering with "clink set terminate_autoanswer 1" I presume?

Would you mind posting the contents of your clink.log after Clink fails to auto-answer the prompt (%userprofile%/appdata/local/clink)?

**Comment by bannmann** [comment]
> Just to be absolutely certain, you have enabled auto-answering with "clink set terminate_autoanswer 1" I presume?

Thanks for asking! Now that I have enabled this setting, the feature works as advertised.

I have to admit that I managed to completely overlook the documentation. Probably, I would have found it had the link on the home page been more prominent. I simply skipped the paragraph with the link because it has a "Extending clink" headline, a topic I am not (yet) interested in.

However, even the documentation does not seem to say that I have to enable this feature. How about listing the default value of each setting, as well?
